superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -164,6 +164,8 @@ const StyledCollapse = styled(Collapse)`
 const StyledTabs = styled(Tabs)`
   .ant-tabs-nav {
     position: sticky;
+    margin-left: ${({ theme }) => theme.gridUnit * -4}px;
```

Comment:
I think there's a better way to fix this, rather than resorting to negative margins. We (I?) can address this in a follow-up if needed, but I believe we can do the following:

* Remove these margin left/right styles here.
* In `FilterTabsContainer`:
  * `& > .ant-tabs-content-holder` -> remove the `padding-right`
  * `&.ant-tabs-left > .ant-tabs-content-holder > .ant-tabs-content > .ant-tabs-tabpane` -> set the `padding-left` to 0 (or remove the line, but then a 24px padding comes into play from somewhere, not sure if we can kill that).
  * add a selector for `.ant-tabs-content-holder` with `padding: 0 ${({ theme }) => theme.gridUnit * 4}px;`
  
Then we'll have fewer styles that are not fighting each other.

Reply:
Thanks for the tip! I'll change it in the next iteration.